### DISCUSSION TITLE
[Nx Plugin] Configure Vitest to use istanbul coverage by default

### DISCUSSION THREAD

**Discussion by yann-combarnous** [nrwl/nx#23026] [General]
I use @nx/react with unitTestRunner set to `vitest`. How do I set default code coverage provider to `istanbul`? Based on https://nx.dev/nx-api/vite/generators/vitest, this is an option from Vitest generator, but not sure where to add it in `nx.json`

**Comment by jaysoo** [comment]
Assuming you are using `@nx/react:app` generator, there isn't an option to set the coverage provider for Vitest. You could run them as separate generators:

```shell
nx g @nx/react:app myapp --unitTestRunner=none
nx g @nx/vite:vitest --project=myapp --coverageProvider=istanbul
```
